docs/en/stack/ml/nlp/ml-nlp-overview.asciidoc

```diff
@@ -4,6 +4,34 @@
 {nlp-cap} (NLP) refers to the way in which we can use software to understand
 natural language in spoken word or written text.
 
+[discrete]
+[[nlp-elastic-stack]]
+== NLP in the {stack}
+
+Elastic offers a wide range of possibilities to leverage natural language
+processing.
+
+You can **integrate NLP models from different providers** such as Cohere,
+HuggingFace, or OpenAI and use them as a service through the 
+{ref}/inference-apis.html[{infer} API]. You can also use <<ml-nlp-elser,ELSER>>
+(the retrieval model trained by Elastic) and <<ml-nlp-e5,E5>> in the same way.
+This {ref}/semantic-search-inference.html[tutorial] walks you through the
+process of using the various services with the {infer} API.
+
+You can **upload and manage NLP models** using the Eland client and the
+<<ml-nlp-deploy-models,{stack}>>. Find the 
+<<ml-nlp-model-ref,list of supported architectures here>>. Refer to <<setup>> to
```

Comment:
Do you want to have a link to `Setup and Security` here in the introduction? What it links to is quite low level details.

Perhaps link to the examples at https://stack-docs_bk_2679.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-nlp-examples.html